thunder/tests/test_transforms.py

```diff
@@ -334,3 +334,41 @@ def get_model():
 
     assert jm_ref._get_shared_names()["0.weight"] == {"0.weight", "4.weight"}
     assert jm._get_shared_names()["0.weight"] == {"0.weight", "4.weight"}
+
+
+def test_lora_transform_linear():
+    from thunder.transforms import LORATransform
+
+    DIM = 512
+
+    class Model(torch.nn.Module):
+        def __init__(self) -> None:
+            super().__init__()
+            self.fc1 = torch.nn.Linear(DIM, DIM)
+            self.fc2 = torch.nn.Linear(DIM, DIM)
+
+        def forward(self, x):
+            x = self.fc1(x)
+            x = torch.nn.functional.relu(x)
+            x = self.fc2(x)
+            return x
+
+    model = Model()
+    x = torch.randn(DIM, DIM)
+
+    loratransform = LORATransform(r=16, lora_alpha=32)
+
+    jmodel = thunder.jit(
+        model,
+        transforms=[
+            loratransform,
+        ],
+    )
+    actual = jmodel(x)
+    original_jmodel = thunder.jit(model)
```

Comment:
I wonder if we could use LitGPT's LoraLinear as a reference, wdyt?

Reply:
I think that'll be much better than the current test case :) thanks for the suggestion!

Reply:
However, as I try to add LitGPT's lora linear, i think there has to be a way to fix the seed right? (when initializing lora_a and lora_b)